.pre-commit-config.yaml

```diff
@@ -1,19 +1,19 @@
 repos:
   - repo: https://github.com/asottile/add-trailing-comma
-    rev: v2.3.0
+    rev: v3.0.1
     hooks:
       - id: add-trailing-comma
   - repo: https://github.com/psf/black
-    rev: 22.10.0
+    rev: 23.7.0
     hooks:
       - id: black
   - repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
     hooks:
       - id: isort
         name: isort (python)
   - repo: https://github.com/pycqa/flake8
-    rev: '5.0.4'
+    rev: '6.1.0'
```

Comment:
@dmytro-kyrychenko Please move changes to this file into separate pull request.
Question: Is there any reason why this version is in quotes and others are not? If not please unify it.

Reply:
okay, I will create separate pr for this
I am not sure why it is in quotes since we don't actually manually change this files, pre-commit system does this automatically
Should I still change this in new PR?